docs/Reference/Errors.md

```diff
@@ -290,6 +290,22 @@ The logger accepts either a `'stream'` or a `'file'` as the destination.
 The logger should have all these methods: `'info'`, `'error'`,
 `'debug'`, `'fatal'`, `'warn'`, `'trace'`, `'child'`.
 
+#### FST_ERR_LOG_INVALID_LOGGER_INSTANCE
+<a id="FST_ERR_LOG_INVALID_LOGGER_INSTANCE"></a>
+
+The loggerInstance only accepts a logger instance, not configuration object. 
+
+#### FST_ERR_LOG_INVALID_LOGGER_CONFIG
+<a id="FST_ERR_LOG_INVALID_LOGGER_CONFIG"></a>
+
+The logger option only accepts a configuration object, not a logger instance. 
+To pass an instance, use `'loggerInstance'` instead.
+
+#### FST_ERR_LOG_INVALID_LOGGER_CONFIG_AND_LOGGER_INSTANCE
```

Comment:
nitpick: I dont like the error code. 
maybe better:

FST_ERR_LOGGER_AND_LOGGER_INSTANCE_PROVIDED